debian/update.sh

```diff
@@ -60,15 +60,20 @@ if [ -n "$PIKERNELMODDIR" ]; then
     fi
 fi
 
+HOST_ARCH="$(uname -m)"
 ARCH=${ARCH:-arm}
 case "$ARCH" in
 arm)
-    CROSS_COMPILE=${CROSS_COMPILE:-arm-linux-gnueabihf-}
+    if ! grep -q -E "armv(6l|7|7l)" <<<"$HOST_ARCH"; then
+        CROSS_COMPILE=arm-linux-gnueabihf-
+    fi
     # CM1=6 CM3=7 CM4=7l (32 bit kernel)
     kernel_versions="6 7 7l"
     ;;
 arm64)
-    CROSS_COMPILE=${CROSS_COMPILE:-aarch64-linux-gnu-}
+    if ! grep -q -E "(armv8|aarch64)" <<<"$HOST_ARCH"; then
```

Comment:
Is output of uname not consistent or why do we need to match armv8 and aarch64?

Reply:
Yes, pure joy of possible architecture mixups: 

> On a 64-bit processor, you'd see a string starting with armv8 (or above) if the uname process itself is a 32-bit process, or aarch64 if it's a 64-bit process. 

Source: https://unix.stackexchange.com/a/136519

Reply:
Yeah, but this would result in a 32-bit userland and thouhs bild a 32bit kernel ...

Reply:
Right. Fixed this and now it's only `aarch64`